src/Zicht/Bundle/MessagesBundle/Translator/GoogleTranslator.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+/**
+ * @copyright Zicht Online <http://zicht.nl>
+ */
+
+namespace Zicht\Bundle\MessagesBundle\Translator;
+
+use Google\Cloud\Translate\V2\TranslateClient;
+
```

Comment:
Misschien hier een `if (!class_exists('\Google\Cloud\Translate\...')) { throw SomeException('Explicit message + instructions'); }`
Ik vind het altijd irritant dat je dus een error krijgt dat een class niet bestaat en dat je dan moet gaan uitzoeken waarom... Kost voor elke dev weer tijd om er achter te komen dat je een bepaalde composer package mist en dus nog moet installeren.

Reply:
Deze wordt nergens als service gedefinieerd, je moet dit zelf gaan doen in je project en 'm daarna injecteren in het command via `setBatchTranslator`. Het is een hulpje om het mogelijk te maken snel met Google aan de slag te gaan maar wordt niet afgedwongen.

Reply:
Dat heb ik inderdaad begrepen, maar dan alsnog...

Reply:
Maar dan gaat er toch verder niets mis en hoef je niets uit te zoeken als je deze update binnen krijgt? Zou jij hier iets aan willen veranderen?